commit 11c123fd4c7c79a365ad3917ceadc98fcde83cc6 (refs/changes/83/4983/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-10-19T21:12:50-07:00 (12 months ago)
    
    TRITON-894 vmapi test suite hanging on vms.changefeed.test.js

diff --git a/test/vms.changefeed.test.js b/test/vms.changefeed.test.js
index d21a727..7c97c36 100644
--- a/test/vms.changefeed.test.js
+++ b/test/vms.changefeed.test.js
@@ -242,6 +242,7 @@ exports.create_vm = function (t) {
         billing_id: '00000000-0000-0000-0000-000000000000',
         ram: 64,
         quota: 10,
+        cpu_cap: 100,
         customer_metadata: md,
         creator_uuid: CUSTOMER,
         role_tags: ['fd48177c-d7c3-11e3-9330-28cfe91a33c9']
